docs/Structure.md

```diff
@@ -0,0 +1,7 @@
+# RichText structure
```

Comment:
I'd rather handle such small portion of information in `README.md` (see also other comments for this file).
Or is there a wider plan for this doc?